common-caching.js

```diff
@@ -2,90 +2,145 @@
 // prevent private cookie cache skipping from working, as it is
 // applied after this worker runs.
 
+// When any cookie in this list is present in the request, cache will be skipped
 const PRIVATE_COOKIES = ["csrftoken", "sessionid"];
 
+// These querystring keys are stripped from the request as they are generally not
+// needed by the origin.
 const STRIP_QUERYSTRING_KEYS = [
-  "utm_source",
-  "utm_campaign",
-  "utm_medium",
-  "utm_term",
-  "utm_content",
-  "gclid",
-  "fbclid",
-  "dm_i", // DotDigital
+    "utm_source",
+    "utm_campaign",
+    "utm_medium",
+    "utm_term",
+    "utm_content",
+    "gclid",
+    "fbclid",
+    "dm_i", // DotDigital
 ];
 
-/**
- * List of cachable HTTP status codes comes from here:
- * https://www.w3.org/Protocols/rfc2616/rfc2616-sec13.html#sec13.4
- */
+// Only these status codes should be considered cacheable
+// (from https://www.w3.org/Protocols/rfc2616/rfc2616-sec13.html#sec13.4)
 const CACHABLE_HTTP_STATUS_CODES = [200, 203, 206, 300, 301, 410];
 
 addEventListener("fetch", (event) => {
-  event.respondWith(main(event));
+    event.respondWith(main(event));
 });
 
 async function main(event) {
-  const cache = caches.default;
-  let request = event.request;
-  request = stripIgnoredQuerystring(request);
-
-  if (!requestIsCachable(request)) {
-    return fetch(request);
-  }
-
-  let response = await cache.match(request);
-  if (!response) {
-    response = await fetch(request);
-    if (responseIsCachable(response)) {
-      event.waitUntil(cache.put(request, response.clone()));
+    const cache = caches.default;
+    let request = event.request;
+    [request, strippedParams] = stripIgnoredQuerystring(request);
```

Comment:
I think you need to have defined `strippedParams` beforehand for this to work.
```suggestion
    let [request, strippedParams] = stripIgnoredQuerystring(event.request);
```